components/purchase_modal/purchase_modal.tsx

```diff
@@ -616,7 +616,7 @@ class PurchaseModal extends React.PureComponent<Props, State> {
             return product.recurring_interval === RecurringIntervals.YEAR && product.sku === CloudProducts.PROFESSIONAL;
         };
 
-        const yearlyProductMonthlyPrice = parseInt(this.state.selectedProductPrice || '0', 10);
+        const yearlyProductMonthlyPrice = (parseInt(this.state.selectedProductPrice || '0', 10) / 12).toFixed(0);
```

Comment:
This causes rounding errors. I think the actual prod yearly price will be $10/user/month, but in test its currently set to $7.50, which makes this get rounded to $8. I think we need to support showing up to two decimal places. I think `intl.formatNumber` can help with that. Closest example I can find is here: https://github.com/mattermost/mattermost-webapp/blob/master/components/self_hosted_purchase_modal/seats_calculator.tsx#L119